docs/guide/machine-learning.asciidoc

```diff
@@ -29,3 +33,48 @@ model in Elasticsearch
 >>> es_model.predict(training_data[0])
 [0 1 1 0 1 0 0 0 1 0]
 ------------------------
+
+[discrete]
+[[ml-nlp-pytorch]]
+=== Natural language processing (NLP) with PyTorch
+
+For NLP tasks, Eland allows importing PyTorch trained BERT models into {es}. 
```

Comment:
```suggestion
For NLP tasks, Eland enables you to import PyTorch trained BERT models into {es}. 
```
I always prefer "enable" to "allow", since the latter implies just giving you permission to do something, whereas the former implies it helps you accomplish the task.